test/Serialization/setup_tests.jl

```diff
@@ -7,18 +7,18 @@ using JSONSchema, Oscar.JSON
 if !isdefined(Main, :test_save_load_roundtrip)
   mrdi_schema = Schema(JSON.parsefile(joinpath(Oscar.oscardir, "data", "schema.json")))
 
-  function test_save_load_roundtrip(func, path, original::T; params=nothing) where {T}
+  function test_save_load_roundtrip(func, path, original::T; params=nothing, with_attrs::Bool=false) where {T}
```

Comment:
maybe rename the kwarg `with_attrs` to something like `save_with_attrs` to underline that this is only used for saving but not for loading?

Reply:
well I just changed it so default behaviour is to load all attributes, so if you save with attributes it will load them all.
If you don't save any then load with behave as usual

Reply:
ok, true. didn't think of that. thanks for the explanation